### DISCUSSION TITLE
Show next 6 weeks, instead of the current month

### DISCUSSION THREAD

**Discussion by ciruz** [wojtekmaj/react-calendar#574] [Q&A] [answered]
Hey,

I got a date range (for ex. ``2021-12-21`` to ``2022-18-01-18``), and I want the calendar to show the next 6 weeks, starting with the current day (for ex. today => ``2021-12-18``).

Default Calendar:

```javascript
<Calendar
    minDate={firstDay} //2021-12-21
    maxDate={lastDay} //2022-18-01
    showDoubleView={true}
    showNavigation={false}
/>
```

My current result:
![image](https://user-images.githubusercontent.com/987947/146643262-bc7080ac-96b6-46de-bcef-f6b00e21502d.png)

But is there perhaps another view (like a week view) that I'm overlooking?

The result I would like to achieve:
![image](https://user-images.githubusercontent.com/987947/146643478-1e25eafe-2382-4d1d-81c4-b5fc24c118cc.png)

Maybe someone has an idea?

Thank you

**Comment by wojtekmaj** [comment, marked as answer]
Use `showFixedNumberOfWeeks` to always show 6 weeks. However, There is no possibility of showing custom date ranges e.g. starting month view mid month. Sorry!

**Reply by R3moulade** [reply]
it would be really cool if this could be implemented, i was also looking for only displaying one week instead of the whole month

**Reply by rfinni** [reply]
I agree. Was looking to implement something similar to your screenshot, basically a rolling 30 day range based on todays date.
